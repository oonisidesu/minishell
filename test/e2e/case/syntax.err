| ls | cat
ls | | cat
ls | cat | cat | | cat
ls | cat |
AA="a a"
> $AA
cat << EOF | cat << EOF2 | | cat << EOF3
