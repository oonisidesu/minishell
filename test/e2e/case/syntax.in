| ls | cat
ls | | cat
ls | cat | cat | | cat
ls | cat |
