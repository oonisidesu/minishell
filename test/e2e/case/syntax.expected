minishell: syntax error near unexpected token `|'
minishell: syntax error near unexpected token `|'
minishell: syntax error near unexpected token `|'
minishell: syntax error near unexpected token `|'
minishell: $AA: ambiguous redirect
minishell: syntax error near unexpected token `|'
